app/src/main/java/ch/epfl/sdp/peakar/user/challenge/NewChallengeItem.java

```diff
@@ -0,0 +1,72 @@
+package ch.epfl.sdp.peakar.user.challenge;
+
+import android.net.Uri;
+
+import androidx.annotation.Nullable;
+
+import java.time.LocalDateTime;
+import java.util.HashMap;
+
+/**
+ * Item holding all information regarding challenge
+ */
+public class NewChallengeItem {
+    private final String name;
+    private final String founderID;
+    private final Uri founderUri;
+    private final int status;
+    private final int numberOfParticipants;
+    private final LocalDateTime startDateTime;
+    private final LocalDateTime endDateTime;
+    private final HashMap<String, Integer> challengeRanking;
+    private final HashMap<String, String> enrolledUsers;
+
+    /**
+     *  @param name challenge name (name of the owner + challenge)
+     * @param numberOfParticipants number of enrolled users
+     * @param startDateTime start time
+     * @param endDateTime end time
+     */
+    public NewChallengeItem(String name, String founderID, Uri founderUri, int status, int numberOfParticipants,
+                            @Nullable LocalDateTime startDateTime, @Nullable LocalDateTime endDateTime
+                            , @Nullable HashMap<String, Integer> challengeRanking,
+                            @Nullable HashMap<String,String> enrolledUsers) {
```

Comment:
Some parameters are missing (I suggest we either put or remove all the parameters descriptions). 
However, I find quite confusing the `name` description. Does it represent the name of the challenge or the name of the owner?
If I call `getName()` on a challenge item, I would expect to receive the name of the challenge.

Reply:
Refactored